config/crd/bases/components.opendatahub.io_codeflares.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.1
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
you may need to delete the bin folder, and re run `make geenrate`

Reply:
Done + the targets you said below